InfiniTAM/ITMLib/Objects/ITMMesh.h

```diff
@@ -49,10 +49,11 @@ namespace ITMLib
 				{
 					for (uint i = 0; i < noTotalTriangles; i++)
 					{
-						fprintf(f, "v %f %f %f\n", triangleArray[i].p0.x, triangleArray[i].p0.y, triangleArray[i].p0.z);
-						fprintf(f, "v %f %f %f\n", triangleArray[i].p1.x, triangleArray[i].p1.y, triangleArray[i].p1.z);
-						fprintf(f, "v %f %f %f\n", triangleArray[i].p2.x, triangleArray[i].p2.y, triangleArray[i].p2.z);
-					}
+                        const auto& clr = triangleArray[i].clr;
```

Comment:
@vinjn: Please can you replace `auto` with `Vector3f`? The former breaks the build on older compilers.